applications/ParticleMechanicsApplication/CMakeLists.txt

```diff
@@ -35,13 +35,19 @@ if(${KRATOS_BUILD_TESTING} MATCHES ON)
   file(GLOB_RECURSE  KRATOS_PARTICLE_MECHANICS_APPLICATION_TESTING_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/tests/cpp_tests/*.cpp)
 endif(${KRATOS_BUILD_TESTING} MATCHES ON)
 
+if((${USE_MPI} MATCHES ON))
+ set(EXTRA_LIBRARIES "KratosMPICore")
```

Comment:
I strongly recommend to avoid this! This will cause lots of issues down the line

You already have a Trilinos extension, all MPI code should go there